.eslintrc.js

```diff
@@ -2,9 +2,17 @@ module.exports = {
   env: {
     es6: true,
   },
-  extends: ["airbnb-base", "prettier"],
+  extends: [
+    "airbnb-base",
+    "airbnb-typescript/base",
+    "prettier",
+    "plugin:jsdoc/recommended",
```

Comment:
it'd be absolutely divine if you explained these changes for my own edification

Reply:
Just here or also in the commit message?

- Extended `airbnb-typescript/base` to get it to stop yelling at me about importing files without file extension. Saw this recommended as the fix [on StackOverflow](https://stackoverflow.com/a/67610259/249801). And it makes sense to me that if we are extending Airbnb's lint rules and using TypeScript, we probably want their TypeScript-specific lint rules, too.

Reply:
Also added these notes to the commit message